rfd/0056-sql-backend.md

```diff
@@ -0,0 +1,110 @@
+---
+authors: Jim Bishopp (jim@goteleport.com)
+state: draft
+---
+
+# RFD 56 - SQL Backend
+
+
+## What
+
+A SQL Backend extends Teleport with the ability to store its data in either
```

Comment:
I added a sentence describing what a backend is.

Reply:
I think it would be good to use the language, core cluster state, and if we're providing the other backends.  That language is here https://goteleport.com/docs/setup/reference/backends/

Reply:
@stevenGravy Added the core cluster state term.